src/modules/dashboard/components/ExpenditurePage/Batch/PreviewDialog/PreviewDialog.css

```diff
@@ -0,0 +1,95 @@
+.wrapper {
+  padding: 10px 0 28px;
+}
+
+.header {
+  font-size: var(--size-medium-l);
+  font-weight: var(--weight-bold);
+  letter-spacing: var(--spacing-medium);
+  columns: var(--dark);
+}
+
+.row {
+  display: flex;
+  align-items: center;
+  border-bottom: 1px solid color-mod(var(--temp-grey-blue-7) alpha(15%));
+}
+
+.borderTop {
+  border-top: 1px solid color-mod(var(--temp-grey-blue-7) alpha(15%));
+}
+
+.lastRow {
+  border-bottom: none;
+}
+
+.cell {
+  padding: 8px 0;
+  width: 33%;
+  font-size: var(--size-normal);
+  text-align: right;
+  color: color-mod(var(--dark) alpha(85%));
+  letter-spacing: 0.1px;
+}
+
+.left {
+  composes: cell;
+  text-align: left;
+}
+
+.left > span {
+  font-size: var(--size-smallish);
+  letter-spacing: var(--spacing-medium);
+}
+
+.validUser > span {
+  color: var(--temp-grey-blue-7);
+}
+
+.middle {
+  composes: cell;
+}
+
+.right {
+  composes: cell;
+}
+
+.tableHeader {
+  display: flex;
+  align-items: center;
+  margin-right: 16px;
+  padding: 0px 0 15px;
+  font-size: var(--size-smallish);
+  font-weight: var(--weight-bold);
+  color: var(--dark);
+  letter-spacing: var(--spacing-medium);
+}
+
+.error {
+  color: var(--danger);
+}
+
+.items {
+  margin-right: 16px;
+  max-height: calc(100vh - 320px);
+  overflow-y: auto;
+
+  &::-webkit-scrollbar {
+    width: 9px;
+    border-radius: var(--radius-normal);
+    background-color: transparent;
+  }
+
+  &::-webkit-scrollbar-track {
+    border-radius: var(--radius-normal);
+    background-color: var(--colony-light-blue);
+  }
+
+  &::-webkit-scrollbar-thumb {
+    width: 5px;
+    border-radius: var(--radius-normal);
+    border-right: 2px solid var(--colony-light-blue);
+    border-left: 2px solid var(--colony-light-blue);
```

Comment:
@agnieszkajarosikloj The scrollbar is looking a little warped for me. 

I tried this instead. Let me know what you think.

``` css
&::-webkit-scrollbar-thumb {
    border-radius: 6px;
    border: 2px solid var(--colony-light-blue);
    background-color: color-mod(var(--temp-grey-blue-7) alpha(65%));
  }
```

Before:
![scrollbar1](https://user-images.githubusercontent.com/64402732/188853366-9c04f074-8ac6-45d6-b14f-6cad4d65691f.png)

After:
![scrollbar2](https://user-images.githubusercontent.com/64402732/188853392-d499d181-b1cc-4cad-9b0e-0cf5224378bb.png)